orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/providers/cf/CloudFoundryServerGroupCreator.java

```diff
@@ -60,6 +62,8 @@
             .build();
     ManifestEvaluator.Result evaluatedManifest = manifestEvaluator.evaluate(stage, manifestContext);
 
+    Moniker moniker = buildMoniker(context);
```

Comment:
I don't see this new variable being used (so the new `buildMoniker` method is also not used)

Reply:
Thanks -- must have missed it when I switched branches and re-did the changes.